bin/babel-node

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env node
 
+var indexOf = require("lodash/array/indexOf");
```

Comment:
Whoa, I do not know what I was thinking about when writing this. Thanks for correcting it.